src/Microsoft.DotNet.Interactive/InstallPackagesMessage.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Formatting;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Html;
+using static Microsoft.DotNet.Interactive.Formatting.PocketViewTags;
+
+namespace Microsoft.DotNet.Interactive
+{
+    // Data bag containing the data to be displayed when reporting nuget resolve progress 
+    public class InstallPackagesMessage
+    {
+        static InstallPackagesMessage ()
+        {
```

Comment:
We shouldn't register formatters in a static constructor because they can be reset and there's no way to get back to this state. It could introduce some annoying test ordering dependencies, among other problems.

Reply:
I will find somewhere to register it.

Reply:
I wonder if we should add a convention, so e.g. if `InstallPackagesMessage` implemented `ITypeFormatter<InstalPackagesMessage>`, we would just use that implementation.

The one issue there would be around MIME types, because it's intended to map to just one: 

https://github.com/dotnet/interactive/blob/54c98c456b7afeba45c599cba8aae4134a97c81e/src/Microsoft.DotNet.Interactive.Formatting/ITypeFormatter.cs#L8-L15